core/src/map.cpp

```diff
@@ -454,7 +454,7 @@ bool Map::update(float _dt) {
     bool tilesLoading = impl->tileManager.hasLoadingTiles();
     bool labelsNeedUpdate = impl->labels.needUpdate();
 
-    if (viewChanged || tilesChanged || tilesLoading || labelsNeedUpdate || impl->sceneLoadTasks > 0) {
+    if (viewChanged || tilesLoading || labelsNeedUpdate || impl->sceneLoadTasks > 0) {
```

Comment:
I am actually not sure about this.
It is possible one of the visible tile is ready which causes `m_tileSetChanged` to be set to `true` (https://github.com/tangrams/tangram-es/blob/1e16d40622633fcaca853999760f856e315fe2d2/core/src/tile/tileManager.cpp#L318). But this does not mean view is complete, as its possible other visible tiles are still loading.

Reply:
See the 'tilesLoading' right next to it - that's what tells if tiles are still loading :)

I've came across this when checking the logs big-scene-refactoring on ios that it didn't show 'view complete' when the view was visibly complete.